shell/platform/android/test/io/flutter/plugin/platform/PlatformPluginTest.java

```diff
@@ -110,4 +114,37 @@ public void platformPlugin_hasStrings() {
     clipboardManager.setPrimaryClip(clip);
     assertFalse(platformPlugin.mPlatformMessageHandler.clipboardHasStrings());
   }
+
+  @Test
+  public void popSystemNavigatorFlutterActivity() {
+    Activity fakeActivity = spy(new Activity());
```

Comment:
prefer using mock (and mocking out any methods the platformplugin needs to use if necessary) so interactions are clear. If you have to make a "real" activity, use Robolectric API explicitly.

Reply:
it's still using spy :) 

`mock` an activity or do something like https://github.com/flutter/engine/blob/master/shell/platform/android/test/io/flutter/embedding/android/FlutterActivityTest.java#L61

Reply:
My mistake, i pushed for testing before finishing all the changes